tribe-ext-eventbrite-addl-opts/index.php

```diff
@@ -52,76 +46,63 @@ public function add_settings() {
-			),
-
 			$this->opts_prefix . 'content_before' => array(
-				'type'				=> 'text',
-				'label'				=> esc_html__( '<h3> text to display Before/Above iframe', 'tribe-extension' ),
-				'tooltip'			=> esc_html__( 'By default there is no content before the iframe. You might want to try something like "Eventbrite Tickets". It will be output as an <h3> heading with "tribe_ext_eventbrite_opts" class.', 'tribe-extension' ),
-				'validation_type'	=> 'textarea', // runs wp_kses()
+				'type'            => 'text',
+				'label'           => esc_html__( 'Heading above ticket area', 'tribe-extension' ),
+				'tooltip'         => esc_html__( 'Adds a heading above the ticket area. By default, this is blank and no heading is shown.', 'tribe-extension' ),
+				'validation_type' => 'textarea',
 			),
 
 			$this->opts_prefix . 'content_after' => array(
-				'type'				=> 'text',
-				'label'				=> esc_html__( '<span> text to display After/Below iframe', 'tribe-extension' ),
-				'tooltip'			=> esc_html__( 'By default there is no content after the iframe. It will be output as a <span> with "tribe_ext_eventbrite_opts" class.', 'tribe-extension' ),
-				'validation_type'	=> 'textarea', // runs wp_kses()
+				'type'            => 'text',
+				'label'           => esc_html__( 'Text underneath ticket area', 'tribe-extension' ),
+				'tooltip'         => esc_html__( 'By default, there is no content after the ticket area.', 'tribe-extension' ),
+				'validation_type' => 'textarea',
 			),
 
-			// see https://central.tri.be/issues/62402
 			$this->opts_prefix . 'iframe_px' => array(
-				'type'				=> 'text',
-				'label'				=> esc_html__( 'Display Tickets iframe Height (pixels)', 'tribe-extension' ),
-				'default'			=> 200, // required else positive_int validation will throw an error each time settings are saved if this field is left blank
-				'tooltip'			=> esc_html__( 'The default is 200.', 'tribe-extension' ),
-				'validation_type'	=> 'positive_int',
+				'type'            => 'text',
+				'label'           => esc_html__( 'Display Tickets iframe Height (pixels)', 'tribe-extension' ),
+				'default'         => 200,
+				'tooltip'         => esc_html__( 'The default is 200.', 'tribe-extension' ),
+				'validation_type' => 'positive_int',
 			),
 
 			$this->opts_prefix . 'locale' => array(
-				'type'				=> 'dropdown',
-				'label'				=> esc_html__( 'Eventbrite Locale', 'tribe-extension' ),
-				'default'			=> 'en_US',
-				'tooltip'			=> sprintf( esc_html__( 'This affects the API call. The default is %sUnited States (en_US)%s.', 'tribe-extension' ), '<strong>', '</strong>' ),
-				'validation_type'	=> 'options',
-				'options'			=> $this->eb_locale_country_name_array(),
+				'type'            => 'dropdown',
+				'label'           => esc_html__( 'Eventbrite Locale', 'tribe-extension' ),
+				'default'         => 'en_US',
+				'tooltip'         => esc_html__( 'This affects the API call.', 'tribe-extension' ),
```

Comment:
I bet a lot of people would not know what an API call is. Maybe we could make this easier to understand? Something like `This tells eventbrite.com which locale to use.`